### DISCUSSION TITLE
Pass PERCY_TOKEN from .env file

### DISCUSSION THREAD

**Discussion by viktorNedelko** [percy/cli#1027] [Support] [answered]
## The problem
PERCY_TOKEN value is not being passed if it is provided within the project .env file. Right now it is required to pass the env variable on the OS level. It would be great if one could also pass this variable from .env file while using [dotenv package](https://www.npmjs.com/package/dotenv).

PERCY_TOKEN within .env file:
<img width="1077" alt="Screenshot 2022-08-09 at 18 49 12" src="https://user-images.githubusercontent.com/95863939/183699067-8e193d3e-73df-4a54-8ab4-7ef08715a1bc.png">
Same token with export statement:
<img width="1066" alt="Screenshot 2022-08-09 at 18 51 43" src="https://user-images.githubusercontent.com/95863939/183699526-53cd21b3-e3c8-410d-aefb-81648f099d1a.png">

Since both those values are reachable from within the project via `process.env.PERCY_TOKEN`, it should be possible to implement such a feature. 

## Environment

- Node version: v16.16.0
- `@percy/cli` version: 1.8.0
- Version of Percy SDK you’re using: 1.8.0
- If needed, a build or snapshot ID: 
- OS version: MacOS Monterey 12.5
- Type of shell command-line [interface]:

**Comment by Robdel12** [comment, marked as answer]
Hey @viktorNedelko! This is supported. I would verify you have the the token set correctly in the `.env` file. I see a space in there, so it might be that.

```
# .env file
PERCY_TOKEN="abc123"
```

**Reply by viktorNedelko** [reply]
Thanks, that helped. :)
